robusta_krr/core/runner.py

```diff
@@ -348,9 +353,64 @@ async def run(self) -> int:
             self._process_result(result)
         except (ClusterNotSpecifiedException, CriticalRunnerException) as e:
             logger.critical(e)
+            publish_error(traceback.format_exc())
             return 1  # Exit with error
         except Exception:
             logger.exception("An unexpected error occurred")
+            publish_error(traceback.format_exc())
             return 1  # Exit with error
         else:
             return 0  # Exit with success
+
+    def _send_result(self, url: str, start_time: datetime, scan_id: str, result: Result):
+        result_dict = json.loads(result.json(indent=2))
+        _send_scan_payload(url, scan_id, start_time, result_dict, is_error=False)
+
+def publish_error(url: str, scan_id: str, start_time: str, error: str):
+    _send_scan_payload(url, scan_id, start_time, error, is_error=True)
+
+def publish_error(error: str):
```

Comment:
Are you sure that this will work? e.g. i don't think that python support method overloading that way.